src/modules/notificationAlert/i18n/nb_NO.root.json

```diff
@@ -3,6 +3,7 @@
 		"danger": "Error message (red)",
 		"info": "Info (blue)",
 		"success": "Confirmation (green)",
+		"unimportant": "Simple (gray)",
```

Comment:
According to merriam webster is grey more popular in british speaking countries.